engine/install/binaries.md

```diff
@@ -173,6 +176,31 @@ The macOS binary includes the Docker client only. It does not include the
     This command downloads a test image and runs it in a container. When the
     container runs, it prints a message and exits.
 
+## Install server and client binaries on Windows
+
+You can install Docker from binaries on Windows Server 2016 or Windows 10. Both
+the `dockerd.exe` and `docker.exe` binaries are included.
+
+1.  Use the following PowerShell commands to install and start Docker:
+
+    ```powershell
+    PS C:\> Invoke-WebRequest https://download.docker.com/win/static/stable/x86_64/docker-{{ site.docker_ce_version }}.zip -UseBasicParsing -OutFile docker.zip
+    PS C:\> Expand-Archive docker.zip -DestinationPath $Env:ProgramFiles
+    PS C:\> Remove-Item -Force docker.zip
+    PS C:\> dockerd --register-service
+    PS C:\> Start-Service docker
+    ```
+
+2.  Verify that Docker is installed correctly by running the `hello-world`
+    image.
+
+    ```powershell
+    PS C:\> docker run hello-world:nanoserver
```

Comment:
Looks like `hello-world:nanoserver` currently supports Windows kernels `10.0.20348.230` and `10.0.17763.2183`, which correspond with Windows Server 2019 and Windows Server 2022, so this will fail on Windows Server 2016 🤔. Wondering if there's another useful image to test that the installation is working, or if we should "ignore" this issue @StefanScherer